### PR TITLE
LIS : bump

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

**Comment by extrowerk** [comment]
@alaviss : afaik the test stuff is ok.

**Comment by extrowerk** [comment]
@alaviss : all the proposed changes implemented in the recipe, but github shows it wrong.

File: sci-libs/lis/lis-2.0.7.recipe [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	$commandBinDir/bin/hpcg_kernel \
+	$commandBinDir/bin/hpcg_spmvtest \
+	$commandBinDir/bin/lsolve \
+	$commandBinDir/bin/spmvtest1 \
+	$commandBinDir/bin/spmvtest2 \
+	$commandBinDir/bin/spmvtest2b \
+	$commandBinDir/bin/spmvtest3 \
+	$commandBinDir/bin/spmvtest3b \
+	$commandBinDir/bin/spmvtest4 \
+	$commandBinDir/bin/spmvtest5
```
Comment (alaviss) [inline sci-libs/lis/lis-2.0.7.recipe]: A bit of sorting and a small fix

```diff
-	$libDir/liblis.so.$libVersion \
-	$commandBinDir/bin/esolve \
-	$commandBinDir/bin/esolver \
-	$commandBinDir/bin/gesolve \
-	$commandBinDir/bin/gesolver \
-	$commandBinDir/bin/hpcg_kernel \
-	$commandBinDir/bin/hpcg_spmvtest \
-	$commandBinDir/bin/lsolve \
-	$commandBinDir/bin/spmvtest1 \
-	$commandBinDir/bin/spmvtest2 \
-	$commandBinDir/bin/spmvtest2b \
-	$commandBinDir/bin/spmvtest3 \
-	$commandBinDir/bin/spmvtest3b \
-	$commandBinDir/bin/spmvtest4 \
-	$commandBinDir/bin/spmvtest5
+	$commandBinDir/esolve \
+	$commandBinDir/esolver \
+	$commandBinDir/gesolve \
+	$commandBinDir/gesolver \
+	$commandBinDir/hpcg_kernel \
+	$commandBinDir/hpcg_spmvtest \
+	$commandBinDir/lsolve \
+	$commandBinDir/spmvtest1 \
+	$commandBinDir/spmvtest2 \
+	$commandBinDir/spmvtest2b \
+	$commandBinDir/spmvtest3 \
+	$commandBinDir/spmvtest3b \
+	$commandBinDir/spmvtest4 \
+	$commandBinDir/spmvtest5 \
+	$libDir/liblis.so.$libVersion

File: sci-libs/lis/lis-2.0.7.recipe [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		--enable-shared
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm $libDir/liblis.la
+	rm -rf $prefix/share
```
Comment (alaviss) [inline sci-libs/lis/lis-2.0.7.recipe]: Why was this folder removed?

**Comment by extrowerk** [comment]
Tested by me

**Comment by Begasus** [comment]
Some people need to get in line here, for some recipes one says to include autoconf and here it's advertised as to not include it ...

**Pull request merged**